### DISCUSSION TITLE
Welcome to IS-Y2-Codes Discussions!

### DISCUSSION THREAD

**Discussion by Kimptu** [Kimptu/IS-Y2-Codes#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kimptu** [comment]
I don't understand why  I can see your files.

**Reply by Mugiraneza-user** [reply]
I don't the problem
